api/services/Search/forPosts.js

```diff
@@ -9,7 +10,7 @@ export default function forPosts (opts) {
 
     // this counts total rows matching the criteria, disregarding limit,
     // which is useful for pagination
-    qb.select(bookshelf.knex.raw('posts.*, count(*) over () as total'))
+    countTotal(qb, 'posts', opts.totalColumnName)
```

Comment:
it was necessary to parameterize the name that `Search` uses for the total column; in the new GraphQL API code it should be the value of `PAGINATION_TOTAL_COLUMN_NAME`, but we have to leave the old column name as default so we don't break the old site.